### DISCUSSION TITLE
从SecretFlow仓库中获取GPU镜像出现报错

### DISCUSSION THREAD

**Discussion by jackcheng1101** [secretflow/secretflow#1483] [Q&A]
docker pull secretflow/secretflow-gpu
Using default tag: latest
Error response from daemon: manifest for secretflow/secretflow-gpu:latest not found: manifest unknown: manifest unknown

**Comment by BrainWH** [comment]
你好，可以尝试加上tag，比如docker pull secretflow/secretflow-gpu:latest,此时会从dockerhub上拉取镜像
如果无法拉取dockerhub镜像出现超时情况，可以尝试官网上的第二种方式[自行构建secretflow的gpu版镜像](https://www.secretflow.org.cn/zh-CN/docs/secretflow/v1.9.0b2/getting_started/installation#option-2-build-the-gpu-docker-image-by-yourself)
